Resources/Prototypes/_DV/Objectives/traitor.yml

```
@@ -152,6 +152,27 @@
     stealGroup: BibleMystagogue
     owner: job-name-rd
 
+- type: entity
+  parent: BaseTraitorObjective
+  id: TraitorCritAnomalyObjective
+  name: Force an anomaly to go super-critical
+  description: Hijack Epistemics's equipment to overload an anomaly and make it explode.
```

No s after possessive `'` on word ending with s
```suggestion
  description: Hijack Epistemics' equipment to overload an anomaly and make it explode.
```